docs/docs/12-context.md

```diff
@@ -145,6 +145,14 @@ void componentDidUpdate(
 )
 ```
 
+Additionally, a new lifecycle method named `componentWillReceiveContext` is included, which is called after `componentWillReceiveProps`, but before the update occurs. This method will *always* be called if contexts are enabled, unlike `componentWillReceiveProps`, which is only called if props change.
```

Comment:
Some of these things are just wrong.  There is no guarantee that `componentWillReceiveProps` is called **only** when props change.  The only guarantee is that `componentWillReceiveProps` will be called if props change.